src/lib/server/prisma/schema.prisma

```diff
@@ -11,15 +11,16 @@ datasource db {
 }
 
 model User {
-  id        String      @id @default(nanoid(16))
-  username  String      @unique
-  email     String      @unique
-  password  String
-  name      String
-  verified  Boolean     @default(false)
-  settings  Json        @default("{}")
-  pastes    Paste[]
-  AuthToken AuthToken[]
+  id          String      @id @default(nanoid(16))
+  username    String      @unique
+  email       String      @unique
+  password    String
+  name        String
+  verified    Boolean     @default(false)
+  settings    Json        @default("{}")
+  pastes      Paste[]
+  AuthToken   AuthToken[]
+  ResetToken  ResetToken?
```

Comment:
Instead of an array like `AuthToken`, I opted for a 1-1 (or 0) relationship between User and ResetToken, to represent that there should only exist 1 password reset opportunity at a time for each User. If a new ResetToken is requested while one already exists, the older one is simply overwritten.